src/manager.wsgi

```diff
@@ -124,6 +124,14 @@ def application(environ, start_response):
                     # caseno is invalid number - do nothing, it can be set later
                     pass
 
+            if "bugzillano" in get:
+                try:
+                    bugzillano = filter(int, set(n.strip() for n in get["bugzillano"][0].replace(";", ",").split(",")))
+                    task.set_bugzillano(bugzillano)
+                except Exception:
```

Comment:
Can this be a specific exception?

Reply:
If you have ideas of how to trip exceptions like this let me know and I can try to test them. I'm not sure exactly what this will catch as the input methods seem protected.  For example, I tried to put a non-number in via the 'manager' web UI and it would not take it.  I also cannot trigger this via retrace-server-task:

$ retrace-server-task set --no-verify --bugzillano 11542602,fred 148364915
Status code 404 was returned with message: 'Bugzilla numbers must be integers; invalid literal for int() with base 10: 'fred''